cmake/CubCompilerHacks.cmake

```diff
@@ -25,7 +25,7 @@ if ("NVCXX" STREQUAL "${CMAKE_CUDA_COMPILER_ID}")
   endif()
 
   set(CMAKE_CXX_COMPILER "${CMAKE_CUDA_COMPILER}")
-  set(CMAKE_CUDA_FLAGS "${CMAKE_CUDA_FLAGS} -stdpar")
+  set(CMAKE_CUDA_FLAGS "${CMAKE_CUDA_FLAGS} -cuda -Mnodaz")
```

Comment:
I now that managed memory is not used in this case, but I can see value in having `-gpu=nomanaged` written verbosely.  This way we don't rely on defaults and make our expectations visible. What do you think?

Reply:
Was the managed memory issue WSL specific or did it affect native linux builds, too?

Reply:
@miscco Can you move these back to the other file? I'd rather keep these properties on a target rather than polluting `CMAKE_CUDA_FLAGS`.

Also, I may be conflating issues with my WSL question above😅 Let me know if I'm not making sense there.

Reply:
@allisonvacanti managed memory + concurrent GPU work + windows (or wsl) = segfault. It shouldn't affect linux builds.

Reply:
Yeah this only affects windows. We might be able to better target that option but I am always a bit hesitant about diverging cmake configurations

Reply:
@allisonvacanti I moved the options back and added a TODO for the managed memory option

Reply:
Thanks! While we're updating this, could you move the `-cuda` option over to the other file, too? That should have been a target property all along.

We should also check whether Thrust needs these changes, too. The build systems are nearly identical, so it should be straightforward.